src/genie/libs/parser/nxos/show_track.py

```diff
@@ -0,0 +1,304 @@
+            },
+        },
+    }
+
+
+# =======================
+# Parser for 'show track'
+# =======================
+class ShowTrack(ShowTrackSchema):
+    """ Parser for 'show track' 'show track {id}'"""
+
+    cli_command = ['show track','show track {track_id}']
+
+    def cli(self, track_id = ' ', output=None):
+        if output is None:
+            if track_id:
+               cmd = self.cli_command[1].format(track_id=track_id)
+            else:
+                cmd = self.cli_command[0]
+
+            output = self.device.execute(cmd)
+
+        #Init vars
+        parsed_dict = {}
+       
+        #Track N 
+        p0 = re.compile(r'Track (?P<track_id>\d+)') 
+
+        # Interface Ethernet1/4 IP Routing
+        # IP Route 10.1.1.1/32 Reachability
+        # IPv6 Route 10:1::1:1/32 Reachability
+        # IP SLA 121 Reachability
+        p1 = re.compile(r'^(?P<type>IP Route|Interface|IP SLA|IPv6 Route|List)\s+((?P<address>[\d.]+(\/\d+))|(?P<name>([\d\w.:]+()\/\d+|loopback[\d]+|\d+)))*\s+((?P<parameter>[\w \-]+))')
+
+        # IP Routing is DOWN
+        # Line Protocol is delayed DOWN (20 secs remaining)
+        p2 = re.compile(r'^(?P<parameter>[\w ]+) +is +(?P<state>UP|DOWN|delayed DOWN|delayed UP)( \((?P<secs_remaining>\d+) secs remaining\))*')
+
+        # 1 changes, last change 00:01:04
+        p3 = re.compile(r'^(?P<change_count>\d+) +changes, +last +change'
+            r' +(?P<last_change>[\d\w:]+)')
+   
+        # Threshold weight up 20 down 10
+        # Threshold percentage up 20% down 10%
+        p4  = re.compile(r'^(?P<parameter>[\w ]+) +up +(?P<threshold_up>[\d]+%*) +down +(?P<threshold_down>[\d]+%*)')
+
+        # Track List  12
+        # VRRPV3 Vlan2 2
+        # HSRP Vlan2 2   
+        p5 = re.compile(r'^(?P<name>[a-zA-Z3]{3,6}) +'
+            r'(?P<interface>[A-Za-z0-9\/.]{3,}) +((?P<group_id>\d+)|(\n))')
+
+        # Delay up 20 secs, down 10 secs
+        # Delay up 10000 milliseconds, down 10000 milliseconds 
+        p6 = re.compile(r'^(Delay )(up (?P<delay_up_secs>[\d]+\s+)secs,*)*(up (?P<delay_up_millisecs>[\d]+\s+)milliseconds,)*(\s*down (?P<delay_down_secs>[\d]+) secs)*( down (?P<delay_down_millisecs>[\d]+) milliseconds)*')
+
+        # object 10 weight 10 UP
+        # object 1 (100%) DOWN
+        p7 = re.compile(r'^(object)\s+(?P<object_id>[\d]+)\s(weight (?P<weight>[\d]+))*((?P<percent>\([\d%]+\)))*\s*(?P<obj_state>[\w \-]+)')
+
+        # VPN Routing/Forwarding table "management"
+        # VPN Routing/Forwarding table "vrf2"
+        p8 = re.compile(r'^(VPN Routing\/Forwarding table )\"(?P<vrf_table>[\w]+)')
+
+        for line in output.splitlines():
+            line = line.strip()
+            
+            #Track n   
+            m = p0.match(line)
+            if m:
+               group = m.groupdict()
+               type_dict1 = parsed_dict.setdefault('track', {})
+               track_id1 = group['track_id']
+               type_dict = type_dict1.setdefault(track_id1,{})
+
```

Comment:
Please add a `continue` here (if applicable)

Reply:
added the 'continue'